src/Handlers/NetworkTestHandler.ts

```diff
@@ -54,7 +54,7 @@ export class NetworkTestHandler
           chunk.indexOf(ReadlinePlatform.HOST_OR_IP_QUESTION) > -1 &&
           config.type === NetworkTestType.TRACEROUTE
         ) {
-          child.stdin.write(`${config.url}${EOL}`);
+          child.stdin.write(`${config.host}${EOL}`);
```

Comment:
Okay, but instead of changing API, you could easily to parse URL to obtain hostname. BTW host is still wrong value for that (we expect a hostname or IP address)